libnopegl/src/backends/gl/hwmap_videotoolbox_darwin_gl.c

```diff
@@ -98,9 +98,9 @@ static int vt_darwin_map_plane(struct hwmap *hwmap, IOSurfaceRef surface, size_t
 
     size_t width = IOSurfaceGetWidthOfPlane(surface, index);
     size_t height = IOSurfaceGetHeightOfPlane(surface, index);
-    if (width > INT_MAX || height > INT_MAX)
+    if (width > UINT32_MAX || height > UINT32_MAX)
```

Comment:
do we really support up to this range? I saw some GL calls in this diff that seem to only accept signed ints. Or in cases such as the mipmap thing.

Reply:
GL mostly use signed only, however, the modern api use  unsigned for dimensions. This check is only there because ngpu dimensions are now unsigned.